api/utils.go

```diff
@@ -299,6 +300,29 @@ func getTransactionsQuery(ctx context.Context, db *sql.DB, offset, limit int, ch
 				}
 				item["blobVersionedHashes"] = bVHashes
 			}
+		case evm.SetCodeTxType:
+			accessList = &evm.AccessList{}
+			rlp.DecodeBytes(accessListRLP, accessList)
+			item["accessList"] = accessList
+			item["chainId"] = uintToHexBig(chainid)
+			item["maxPriorityFeePerGas"] = bytesToHexBig(gasTipCapBytes)
+			item["maxFeePerGas"] = bytesToHexBig(gasFeeCapBytes)
+			item["yParity"] = uintToHexBig(v)			
+			item["maxFeePerBlobGas"] = bytesToHexBig(blobGasFeeBytes)
+			if len(bVHashesRLP) > 0 {
```

Comment:
SetCodeTxType transactions cannot have blob hashes. That would mean that this is always going to return false and the rest doesn't happen, so it shouldn't cause any problems,